tests/js/spec/views/organizationGroupDetails/groupEventDetailsContainer.spec.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import OrganizationEnvironmentsStore from 'app/stores/organizationEnvironmentsStore';
+import GroupEventDetailsContainer from 'app/views/organizationGroupDetails/groupEventDetailsContainer';
+
+jest.mock('app/views/organizationGroupDetails/groupEventDetails', () => ({
+  GroupEventDetails: () => <div>GroupEventDetails</div>,
+}));
+
+describe('groupEventDetailsContainer', () => {
+  const org = TestStubs.Organization();
+  const globalSelection = TestStubs.GlobalSelection();
+
+  beforeEach(() => {
+    OrganizationEnvironmentsStore.init();
+  });
+
+  it('fetches environments', async function() {
+    MockApiClient.addMockResponse({
+      url: `/organizations/${org.slug}/environments/`,
+      body: [{id: 'id', name: 'name'}],
+    });
+    const wrapper = mount(
+      <GroupEventDetailsContainer
+        api={new MockApiClient()}
```

Comment:
I think once you shuffle things around I suspect you won't need this since the container should be wrapped with `withApi` and the other HoCs